lib/features.json

```diff
@@ -258,5 +258,10 @@
     "name": "lessCharsInChat",
     "description": "max chars in chat",
     "versions": ["1.8", "1.10.2"]
+  },
+  {
+    "name": "usesPayload",
```

Comment:
can you make the name of this more specific to statistics like `statisticsUsesPayload`

Reply:
This is a general name in 1.8 so

Reply:
you are right ill make it more specific

Reply:
it's not, what usespayload

Reply:
got merged